### ISSUE TITLE
[FEATURE REQUEST] Display App Version on Home Page with One-Click Update Option

### ISSUE THREAD

**Issue by lirexxx** [pinokiocomputer/pinokio#229]
Hello,

I noticed that on the Home page of Pinokio, while I can see my installed apps, the version of each installed app isn't displayed. Currently, to verify if I have the latest version of an app, I need to click on the app and then navigate further to the Update section.

Since a key principle of Pinokio is to make actions fast and easy with minimal clicks, I propose adding the app version directly on the Home screen, similar to how it's displayed in the "Discover" window. Additionally, it would be great to have a one-click update option. For example, if an update is available for an app, a small, intuitive icon (like a green downward arrow) could appear next to the app. Clicking this icon would immediately take the user to the Update screen, streamlining the process to initiate the update.

I believe this feature would significantly enhance the user experience by making it more efficient and user-friendly, aligning well with Pinokio's core values of simplicity and speed.

Looking forward to your thoughts on this suggestion!

**Comment by ETCore7** [comment]
Agree. Also, update should be optional. CogStudio worked well, but now interpolation is gone and there is an extra column with rendered videos. Newly videos look bad. No way to retry prompts & seeds since those are not saved.
